build.gradle

```diff
@@ -54,17 +55,17 @@ compileJava {
 }
 
 dependencies {
-    compile('org.seleniumhq.selenium:selenium-java:3.4.0'){
+    compile('org.seleniumhq.selenium:selenium-java:3.5.1'){
```

Comment:
How about this solution?
![seleenium](https://user-images.githubusercontent.com/7767781/29572372-726bf712-875c-11e7-9f7f-72181923726f.png)